aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -714,7 +714,7 @@ Tensor dropout_nested(const Tensor& input, double p, bool train) {
   Tensor output_buffer = at::dropout(input_buffer, p, train);
   // regular tensor dropout reuses input size and stride
   // i.e. if input is not contiguous, then output is also discontiguous
-  return wrap_buffer(output_buffer, sizemat.clone(), stridemat.clone(), offsets);
+  return wrap_buffer(output_buffer, sizemat.clone(), stridemat.clone(), std::vector<int64_t>(offsets));
```

Comment:
Why do you need to repack this manually?

Reply:
ohh was just copying along, so just `wrap_buffer(output_buffer, sizemat.clone()`

Reply:
Actually this is because dropout can run on non-contiguous nested tensors so this to make sure the returned tensor has the same discontinuity